bark_ml/model_loader/ModelLoader.cpp

```diff
@@ -0,0 +1,99 @@
+#include "libtensorflow/include/tensorflow/c/c_api.h"
+#include "ModelLoader.hpp"
+#include <cstdlib>
+#include <fstream>
+#include <iostream>
+#include <vector>
+#include <numeric>
+#include <string.h>
+
+void NoOpDeallocator(void* data, size_t a, void* b) {}
+
+ModelLoader::ModelLoader(const char* saved_model_dir)
```

Comment:
please use formatting as described in: https://github.com/bark-simulator/bark-ml/blob/master/STYLE_GUIDE.md